.travis.yml

```diff
@@ -111,6 +111,8 @@ before_install:
     - ${KLEE_SRC}/.travis/install-tcmalloc.sh
     # Install lit (llvm-lit is not available)
     - sudo pip install lit
+    # Install tabulate for klee-stats
+    - sudo pip install tabulate
```

Comment:
@delcypher , the .travis.yml script is supposed to install tabulate here, what did go wrong?